.github/run_browserstack_acceptance.sh

```diff
@@ -6,4 +6,12 @@ export BROWSERSTACK_BUILD_ID="${GITHUB_BRANCH} - ${GITHUB_RUN_ID}"
 COMMIT_MSG_TITLE=$(git log -n 1 --pretty=format:%s)
 export BROWSERSTACK_TEST_RUN_NAME=$COMMIT_MSG_TITLE
 
-npm run acceptance -- --browsers browserstack:ie@11.0 browserstack:safari browserstack:firefox
+if [[ $GITHUB_BRANCH == release/*
```

Comment:
Just curious did you test this bash script somehow? My golden rule of bash scripting: "If you didn't test it, it doesn't work"

Reply:
yup. Tested locally and through circleCI from this pr. I will update the test steps in the description or the pr!

Reply:
sounds good!